src/ExceptionBag.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace WtfPhp\JsonApiErrors;
+
+use Illuminate\Support\Collection;
+use Throwable;
+
+class ExceptionBag implements ExceptionBagInterface
+{
+    protected Collection $bag;
+
+    public function __construct()
+    {
+        $this->bag = new Collection();
+    }
+
+    public function add(Throwable $throwable): void
+    {
+        $this->bag->add($throwable);
+    }
+
+    public function addMultiple(Collection $throwables): void
```

Comment:
I would just use an array here for the input parameter. Otherwise you are forcing the user to create a collection object beforehand. We could simply convert the array to the collection inside the function.

Reply:
Good point, we will change that!